src/content/blog/community-day-2023.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: "Building a community of builders"
+description: "Today, we launch our newest initiative to recognize and celebrate people in our community who go above and beyond, including our very own maintainers."
+publishDate: "December 7, 2023"
+coverImage: "/src/content/blog/_images/community-day-2023/header.webp"
+socialImage: "/src/content/blog/_images/community-day-2023/og.webp"
+lang: "en"
+authors:
+  - sarah
+  - elian
+---
+
+import CommunityProject from '../../components/CommunityProject.astro'
+
+Astro continues to evolve how it works with and for its community. 
+
+With generous Open Collective funding support, we regularly recognize and reward people in our community who made a difference that month: submitting code and bug fixes, being welcoming and helpful in support threads, translating the docs, and other contributions that enrich everyone's Astro experience.
+
+Previously a quarterly event, Astro kicked it into high gear over the last few months and committed to issuing awards on a monthly basis. We also changed the nomination structure from peer-nominated awards to core-maintainer-nominated awards.
+
+This might sound less democratic, but our motivations were to **ensure that our core maintainers were actively on the lookout for the next crop of up-and-comers**. We now hold monthly maintainer meetings entirely dedicated to thinking about and shouting out who is most active in our community. As a core team, we dedicate time to our open-source community's health, growth, and sustainability.
```

Comment:
```suggestion
This might sound less democratic, but our motivations were to **ensure that our core maintainers were actively on the lookout for the next crop of up-and-comers**. We now hold monthly meetings entirely dedicated to reflecting on our contributors. As a core team, we dedicate time to our open-source community's health, growth, and sustainability.
```